### DISCUSSION TITLE
Use Premium when Factory reset

### DISCUSSION THREAD

**Discussion by Onewpus** [tribalfs/GalaxyMaxHzPub#227] [Q&A]
I want to ask the developer, I bought and used the premium app, when I factory reset my samsung phone did the premium feature of the app lose?

**Comment by tribalfs** [comment]
You can restore the premium version by (re)entering the activation code that was provided for your device.

**Comment by H4ll34** [comment]
Same problem here but i take  the code  on playstore of purchased  in didnt works
